modules/programs/zsh.nix

```diff
@@ -252,6 +253,13 @@ in
         default = {};
         description = "Options to configure oh-my-zsh.";
       };
+
+      extraLocalVars = mkOption {
```

Comment:
Since we already have `cfg.sessionVariables` I think it would be more consistent to call it `cfg.localVariables`

Reply:
Done.